src/libs/ajax.js

```diff
@@ -342,7 +342,7 @@ export const DataSet = {
   deleteDataset: async (datasetObjectId, dacUserId) => {
     const url = `${await Config.getApiUrl()}/dataset/${datasetObjectId}/${dacUserId}`;
     const res = await fetchOk(url, _.mergeAll([Config.authOpts(), { method: 'DELETE' }]));
-    return await res.json();
+    return await res;
```

Comment:
this returns a whole response object and should be handled properly : resp.status may be any 200 , 401, 403, 500 . It seems it is not being handled properly where used.